app/views/events/_applicants_overview.html.erb

```diff
@@ -53,11 +53,8 @@
 
   <div class="container-fluid no-padding">
     <div class="btn-group" role="group">
-      <%= link_to t('.accept_all'),
-          events_path, :class => 'btn btn-default'%>
-
-      <%= link_to t('.print_all'),
-                  print_applications_event_path(@event.id), :target => "_blank", :class => 'btn btn-default'%>
+      <%= link_to t('.accept_all'), event_accept_all_applicants_path(@event.id), :class => 'btn btn-default' %>
```

Comment:
Then we both have he same understanding of it, and to_param seems the better/more flexible way to me.
But you are right that its pretty inconsistent, so this is maybe something the scrum masters/all teams should agree on? @Baschdl